server/ColorDeconvolution/ColorDeconvolution.xml

```diff
@@ -60,5 +60,42 @@
       <longflag>stainColor_3</longflag>
       <default>0,0,0</default>
     </double-vector>
+    <boolean>
+      <name>largeImage</name>
+      <label>Large Image?</label>
+      <longflag>largeImage</longflag>
+      <description>True if the input image is a large image that should be handled with large_image</description>
+    </boolean>
+    <integer-vector>
+      <name>region</name>
+      <label>Region</label>
+      <longflag>region</longflag>
+      <description>left,top,width,height of the region of interest</description>
+    </integer-vector>
+    <!-- And now the output parameters -->
+    <integer-vector>
+      <name>region</name>
+      <label>Region</label>
+      <channel>output</channel>
+      <description>left,top,width,height of the region of interest</description>
+    </integer-vector>
```

Comment:
Use the region type here too. Need to test this end-to-end over the web to see if this gets passed along or not. I would suggest drop this until Dave adds writeImage support to large_image that abstracts passing in the location. We can then make a PR.

@manthey @jbeezley What are your thoughts on this?